build.yaml

```diff
@@ -554,7 +553,7 @@ job_type:
       clean_repo: true
 
     run_trial_for_cinder_storage_driver_on_Ubuntu_trusty:
-      on_nodes_with_labels: 'rackspace-ubuntu14-jenkins-slave-DFW-standard-4'
+      on_nodes_with_labels: 'rackspace-jenkins-slave-ubuntu14-standard-4-dfw'
```

Comment:
they do lack a standard

as we move to multi-region across different cloud providers we can tackle that
<provider><os><version><region><vm-type>  

the EC2-plugin requires different AMIs for capping instance numbers across different VM-types.
If we were to use the jclouds plugin for EC2 as we do for Rackspace we could possibly get rid of the 'vm-type'